src/GraphQL.php

```diff
@@ -26,7 +26,6 @@
 use Nuwave\Lighthouse\Events\StartExecution;
 use Nuwave\Lighthouse\Events\StartOperationOrOperations;
 use Nuwave\Lighthouse\Execution\BatchLoader\BatchLoaderRegistry;
-use Nuwave\Lighthouse\Execution\DataLoader\BatchLoader;
```

Comment:
This class should be deleted entirely.

Reply:
Also remove these subsequent classes?
* `Tests\Utils\BatchLoaders\UserLoader`
* `Tests\Integration\Execution\DataLoader\RelationBatchLoaderTest`
* `Tests\Integration\Execution\DataLoader\RelationCountBatchLoaderTest`

Reply:
Only `UserLoader`, the tests cover `RelationBatchLoader` which is kept.

Reply:
The reason I asked is because `UserLoader` is referenced and used in both those test classes. So what to do about that? Remove the test methods which use the `UserLoader`?